markdown/assignments/pool_p/projekt/index.md

```diff
@@ -0,0 +1,31 @@
+---
+type: assignment
+title: "Projekt"
+author: "Andre Matutat (FH Bielefeld)"
+points: 20
+weight: 1
+hidden: true
+---
+
+## Projekt
+
+Während des Projektes sollen Sie sich ein größeres Feature für Ihren Dungeon überlegen, konzeptionieren und implementieren.
+
+Ein Projekt hat eine Laufzeit von **zwei** Wochen. In dieser Zeit bearbeiten Sie nur die Aufgaben aus dem `[Pool K]({{<ref "/assignments/pool_k" >}})`{=markdown} und Ihr Projekt. Andere Pool-Aufgaben müssen Sie nicht parallel bearbeiten.
+
+Sie können insgesamt zwei Projekte durchführen, müssen aber mindestens eines durchführen.
+
+Um den `[Pool P]({{<ref "/assignments/pool_p" >}})`{=markdown} erfolgreich zu absolvieren, müssen sie hier **mindestens 15 Punkte** erreichen.
+
+Sie können die Projekte erst ab Woche 9 (d.h. ab KW22, also in der Woche nach dem ersten E-Assessment) starten.
+
+Sie müssen das Projekt vorher mit ??? absprechen. Bereiten Sie dafür ein kurzes Konzept vor und erklären Sie, was Sie machen möchten, wie Sie dabei vorgehen wollen und welche Probleme/Fragen es gibt. Melden Sie sich bitte frühzeitig.
+
+Nachdem Ihre Projekt-Idee genehmigt wurde, beginnt die Bearbeitungszeit.
+
+Nach der ersten Woche müssen Sie ein schriftliches Feinkonzept und einen ersten Grobentwurf der Implementierung vorstellen.
+
+Nach der zweiten Woche stellen Sie das Ergebnis vor.
+
+Bei der Punktevergabe kommt es nicht nur darauf an, ob Sie Ihr Ziel erreicht haben, sondern auch darauf, ob Sie die Ihnen bekannten Maßnahmen zur Einhaltung der Softwarequalität genutzt haben (Testfälle, Logging, Code-Style). Auch die Qualität Ihrer schriftlichen Ausarbeitung fließt in die Punktevergabe mit ein.
```

Comment:
- zur "schriftlichen Ausarbeitung" fehlen noch die Kriterien: Was soll aufgeschrieben werden, in welchem Umfang ...